docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,86 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+`SHOW TABLES` statement returns all the tables for an optionally specified `database`.
+Additionally, the output of this statement may be filtered via an optional matching
+pattern. If no database is specified then the tables are returned from the 
+current database.
+
+### Syntax
+{% highlight sql %}
+SHOW TABLES [{FROM|IN} database_name] [LIKE 'regex_pattern']
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>{FROM|IN} database_name</em></code></dt>
+  <dd>
+     Specifies the `database` name from which tables are listed.
+  </dd>
+  <dt><code><em>LIKE 'regex_pattern'</em></code></dt>
+  <dd>
+     Specifies the regex pattern that is used to filter out unwanted tables.
+       <br> - The pattern is a regex except that `*` and `|`characters
```

Comment:
@shivusondur drop the "that" ? 
@srowen Can you help check the above sentence please ?

Reply:
@dilipbiswal 
done